spyderlib/plugins/configdialog.py

```diff
@@ -704,6 +752,13 @@ def setup_page(self):
         margins_layout.addWidget(margin_spin)
         prompt_box = newcb(_("Prompt when exiting"), 'prompt_on_exit')
 
+        # Language chooser
+        langs = get_available_translations()
+        choices = list(zip([LANGUAGE_CODES[key] for key in langs], langs))

```

Comment:
I could just use the `LANGUAGE_CODES` variable and remove the call to `langs = get_available_translations()`